Versioning through go build -ldflags -X in template instead of pulling it from source control.
--- a/hscontrol/types/version.go
+++ b/hscontrol/types/version.go
@@ -22,6 +22,10 @@ type VersionInfo struct {
 	Dirty     bool   `json:"dirty"`
 }
 
+var (
+  Version = ""
+)
+
 func (v *VersionInfo) String() string {
 	var sb strings.Builder
 
@@ -44,7 +48,7 @@ var buildInfo = sync.OnceValues(func() (*debug.BuildInfo, bool) {
 
 var GetVersionInfo = sync.OnceValue(func() *VersionInfo {
 	info := &VersionInfo{
-		Version:   "dev",
+		Version:   Version,
 		Commit:    "unknown",
 		BuildTime: "unknown",
 		Go: GoInfo{
